cmd/internal/cli/build.go

```diff
@@ -40,8 +40,37 @@ var (
 	dockerPassword string
 	dockerLogin    bool
 	noCleanUp      bool
+
+	// AllowUnauthenticatedBuild When true, will never ask if you want to build
+	// a container from a unsigned base container. This only works when your bootstrap
+	// is the library
+	AllowUnauthenticatedBuild bool
+	// LocalVerifyBuild if true, ...
+	LocalVerifyBuild bool
 )
 
+// --allow-unsigned, -U
+var buildAllowUnsignedFlag = cmdline.Flag{
+	ID:           "buildAllowUnsignedFlag",
+	Value:        &AllowUnauthenticatedBuild,
+	DefaultValue: false,
+	Name:         "allow-unsigned",
+	ShortHand:    "U",
+	Usage:        "never prompt when building from a unsigned container, only when bootstrap is library",
+	EnvKeys:      []string{"ALLOW_UNSIGNED"},
+}
+
+// --local, -l
+var buildLocalVerifyFlag = cmdline.Flag{
+	ID:           "buildLocalVerifyFlag",
+	Value:        &LocalVerifyBuild,
+	DefaultValue: false,
+	Name:         "local",
+	ShortHand:    "l",
+	Usage:        "nothing here... TODO:",
+	EnvKeys:      []string{"LOCAL_VERIFY_FLAG"}, // <-- TODO:
```

Comment:
what's the TODO?

What does this flag do? LocalVerifyBuild does not tell me much, nor does the `--local` flag itself.